src/util/generateEmbedModules.sh

```diff
@@ -19,14 +19,14 @@ declare -a opts=(
 rm ../modules/modules.inc 2> /dev/null
 touch ../modules/modules.inc
 
-rm ../modules/modulemap.inc 2> /dev/null
+rm ../modules/modulemap.c.inc 2> /dev/null
```

Comment:
This typo malformed the map includes file somewhat since it wasn't being deleted between `make`s.